vignettes/RUVIIIPRPS_TCGA_BrCa_RNAseq.Rmd

```diff
@@ -573,17 +869,36 @@ brca.se.obj <- findNCGsSupervised(
     assess.variables = FALSE, 
     remove.na = 'none', 
     save.se.obj = TRUE, 
+    output.name = 'AnoCorrAs',
     verbose = TRUE)
-brca.se.obj@metadata$NCG$supervised$
 ```
 
+
+### Un supervised 
 ```{r}
-brca.se.obj <- createSupervisedPRPS(
+brca.se.obj <- findNcgsUnSupervised(
     se.obj = brca.se.obj, 
-    assay.name = 'unstranded', 
+    assay.name = 'RawCount', 
+     ncg.selection.method ='AbsNoneOverlap',
+    grid.nb = 1,
+    top.rank.bio.genes = 80,
+    top.rank.uv.genes = 10,
+    uv.variables = c('library.size', 'batch.FcCh', 'tumour.purity.estimate'),
+    variables.to.assess.ncg = c('PAM50','library.size', 'batch.FcCh', 'tumour.purity.estimate'))
+brca.se.obj@metadata$NCG$un.supervised$
+```
+
+
+# Find negative control genes
+## supervised
+
+```{r}
+brca.se.obj <- createPrPsSupervised(
```

Comment:
It seems that currently the explanation for this argument `apply.other.uv.variables` is not in line wit the default value of the argument (TRUE instead of NULL). 
"String of the label of (a) categorical or continuous variable(s) used to define homogeneous groups of samples such as library size, plates from colData(se). By default it is set to 'NULL' meaning the samples will be assigned to homogeneous biological groups of samples using the 'bio.variables'."